backend/capellacollab/sessions/operators/k8s.py

```diff
@@ -109,8 +125,10 @@ def start_persistent_session(
                 "RMT_PASSWORD": password,
                 "FILESERVICE_PASSWORD": password,
                 "T4C_USERNAME": username,
+                "PURE_VARIANTS_LICENSE_SERVER": pure_variants_license_server,
```

Comment:
Shouldn't this dict be replaced by environment?

Reply:
Yes, absolutely.